game/game.wren

```diff
@@ -53,7 +53,7 @@ class Main {
         __timer = 0
 
         // Load Map
-        engine.LoadModel("assets/models/graveyard_level.glb", false)
+        engine.LoadModel("assets/models/graveyard_level_no_landscape.glb", false)
```

Comment:
Have to resolve this, so we use the proper model, maybe we can just overwrite it